### PR TITLE
Feat/socket frontend

### PR DESCRIPTION
**Pull request by Michi4**

### CODE REVIEW

File: LuminAi/src/main/resources/drivers/energy_data.json [hunk 1]
```diff
@@ -1,25 +1,25 @@
 {
-  "devices": [
```
Comment (Stenz123) [inline LuminAi/src/main/resources/drivers/energy_data.json:2]: Do we still need this file?

File: LuminAi/src/main/frontend/src/services/PowerService.js [hunk 2]
```diff
@@ -1,14 +1,21 @@
...
-        store.deviceData = await response.json()
-        console.log(store.deviceData)
-    } catch (error) {
-        console.error('Error fetching data:', error.message);
-    }
-}
\ No newline at end of file
+import { store } from "@/store/Store";
+
+const socket = new WebSocket('ws://localhost:8080/subscribeUpdates');
```
Comment (Stenz123) [inline LuminAi/src/main/frontend/src/services/PowerService.js:3]: please add this to the properties

**Review by Stenz123 — CHANGES_REQUESTED** [review]

File: LuminAi/src/main/resources/drivers/energy_data.json (hunk 1 above)
Comment (Michi4) [reply LuminAi/src/main/resources/drivers/energy_data.json:2]: yes

File: LuminAi/src/main/frontend/src/services/PowerService.js (hunk 2 above)
Comment (Michi4) [reply LuminAi/src/main/frontend/src/services/PowerService.js:3]: 67aabeb

**Review by Stenz123 — APPROVED** [review]

**Pull request merged**